projects/stretch_ovmm/requirements.txt

```diff
@@ -17,4 +17,4 @@ fasttext
 scikit-learn
 lvis
 nltk
-git+https://github.com/openai/CLIP.git
\ No newline at end of file
+git+https://github.com/openai/CLIP.git
```

Comment:
i think we should move these into the main environment.yaml files too

Reply:
The main environment.yaml already has openai-clip. @ykarmesh do you remember why this was required here?

Reply:
we should remove this?